shopinvader_schema_invoice/schemas/amount.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from extendable_pydantic import StrictExtendableBaseModel
+
+from odoo.tools.float_utils import float_round
+
+
+class InvoiceAmount(StrictExtendableBaseModel, extra="ignore"):
+    amount_tax: float | None
+    amount_untaxed: float | None
+    amount_total: float | None
+    amount_due: float | None
+
+    @classmethod
+    def from_record(cls, odoo_rec):
```

Comment:
Feom which odoo model?
```suggestion
    def from_account_move(cls, odoo_rec):
```